libsplinter/src/admin/service/mod.rs

```diff
@@ -1089,6 +1090,214 @@ mod tests {
         mesh.shutdown_signaler().shutdown();
     }
 
+    /// Test that a circuit disband sends the appropriate messages.
```

Comment:
Can you expand this description so it explains the test more.

Reply:
same for other tests added

Reply:
Yep!